### ISSUE TITLE
Display the inverse connection as well

### ISSUE THREAD

**Issue by thigg** [juliuste/bahn.guru#16]
It would be nice, if one could display the inverse connection at the same time.

Use case is:
I want to go to Berlin from Frankfurt sometime in the next month for three days.

Show me at one gasp, what times have good conditions.

A simple solution is to simply show it as a second graphic, or to insert it into the current one.

**Comment by johanngoltz** [comment]
I implemented something like this using a bar chart:
![image](https://user-images.githubusercontent.com/6007658/55473719-9f61dd80-560f-11e9-86c8-e94d3ab86abb.png)
Each bar represents the cost of a three-day roundtrip, e. g. leaving on 12/4 and returning on 15/4 would cost 98€ in total.
You can also see it [in action](https://bahnguru-mit-preisgrafik.cfapps.us30.hana.ondemand.com/graph?origin=Berlin&destination=München&submit=Suchen&class=2&bc=0&departureAfter=&arrivalBefore=&duration=&maxChanges=) (should be online for at least a month). The code is over at [johanngoltz/bahn.guru](https://github.com/johanngoltz/bahn.guru).

I'd love to hear your suggestions and feedback.

**Comment by thigg** [comment]
This is cool. Maybe make the trip length a parameter for the search?

**Comment by johanngoltz** [comment]
Yep, that's on my to-do list. Other things that still need to be done:

- Make this view reachable from the start page, not just by modifying the URL
- Navigation to the day view, just like what happens when you click a day in the original calendar.
- Separate parameters (duration, departure/arrival time, ...) for the in- and the outbound journey would be nice.
- Lastly, I'm sure the CSS I wrote could use some improvement

Paging @juliuste

**Comment by thigg** [comment]
bump. updated my previous comment

**Comment by juliuste** [comment]
>  You can also see it in action (should be online for at least a month). The code is over at johanngoltz/bahn.guru.

Hey, I'm sorry for not answering earlier, the screenshot looks really great, however the link to the demo doesn't seem to work anymore :S

**Comment by johanngoltz** [comment]
Whoops. It's up again.
